.eslintrc.json

```diff
@@ -5,7 +5,7 @@
     "node": true
   },
   "parserOptions": {
-    "ecmaVersion": 2018,
+    "ecmaVersion": 2022,
```

Comment:
This is to get private class field support: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
We no longer need the private class field but this seems fine to have 🤷 

We're targeting modern browser versions anyway ⏩